guideforms.py

```diff
@@ -329,6 +329,13 @@
          'should have "Type=Feature" set and be world readable. '
          'Note: This field only accepts one URL.')),
 
+    'launch_bug_url': forms.URLField(
+        required=False, label='Launch bug URL',
+        help_text=(
+            'Launch bug url (https://bugs.chromium.org/...) to track launch '
+            'aprpovals. '
+            'This bug should be created using a "Launch" issue template.')),
```

Comment:
```suggestion
            'Create this bug using the "Launch" issue template.')),
```

Reply:
Can we link to this template?

Reply:
https://bugs.chromium.org/p/chromium/issues/entry?template=Chrome+Launch+Feature

And potentially add a TODO to "Create" this for them.

Reply:
Done.